Traceback (most recent call last):
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 24, in <module>
    main()
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 21, in main
    return foo6(array, 50)
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 17, in foo6
    return foo5(array, index // 3)
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 15, in foo5
    return foo4(array, index + 20) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 13, in foo4
    return foo3(array, index * 137 ) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 11, in foo3
    return foo2(array, index -1) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 9, in foo2
    return foo1(array, index + 137) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 7, in foo1
    return foo(array, index * 3) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 5, in foo
    return dangerous(array, index) 
  File "/home/qc/Master-Thesis/multilang-stacktrace/python/main.py", line 2, in dangerous
    return array[index + 2]
IndexError: list index out of range
